### ISSUE TITLE
unreal_temp_path not working

### ISSUE THREAD

**Issue by varomixai** [sideeffects/HoudiniEngineForUnreal-v2#125]
Hi
trying to change the unreal_temp_path and I can see the path reflected in the field in the UI in UE4, but has no effect, the temp mesh is still been created in the default location.

![image](https://user-images.githubusercontent.com/47221403/112335143-98f82080-8c81-11eb-8f6a-b2093ff5150c.png)

**Comment by eetumartola** [comment]
Yes this is annoying. Neither the UI parameter or the Houdini attribute seem to have any effect on the location of the temp files.

**Comment by triffski** [comment]
**unreal_bake_folder** doesn't work either. The default from settings gets ignored, the attribute is also ignored and UE5 crashes if you attempt to bake to BP without first manually entering a valid path in the asset's Details panel.

"The Bake Folder (the folder where the baked assets will be stored) is determined by first checking for the "unreal_bake_folder" attribute on the asset (prim, detail). If this not set then the property in the details panel is used. If the property is also not set, then the default bake folder configured in the plugin settings is used." https://www.sidefx.com/docs/unreal/_packaging.html
